server/api/collections.js

```diff
@@ -49,6 +55,27 @@ router.post("collections.create", auth(), async (ctx) => {
   const user = ctx.state.user;
   authorize(user, "create", Collection);
 
+  const collections = await Collection.findAll({
+    where: { teamId: user.teamId, deletedAt: null },
+    attributes: ["id", "index", "updatedAt"],
+    order: [
```

Comment:
We only need the first collection, right?

```suggestion
    limit: 1,
    order: [
```